### PR TITLE
Make borderRadius and borderWidth animatable when using object values.

### PR DESCRIPTION
**Pull request by eelco**

### CODE REVIEW

File: extras/Studio.framer/framer/version [hunk 1]
```diff
@@ -1 +1 @@
-7
\ No newline at end of file
+8
```
Comment (nvh) [inline extras/Studio.framer/framer/version]: Is it necessary to bump the version? Isn't this backwards compatible?

File: test/tests/LayerAnimationTest.coffee [hunk 2]
```diff
@@ -879,3 +879,73 @@ describe "LayerAnimation", ->
...
+
+	describe "Border animations", (done) ->
+
+		it "should animate borders from number to number", (done) ->
+			layer = new Layer
+				borderWidth: 10
+				borderRadius: 20
+			layer.on Events.AnimationEnd, ->
+				layer.borderWidth.should.equal 30
+				layer.borderRadius.should.equal 40
```
Comment (nvh) [inline test/tests/LayerAnimationTest.coffee]: I understand the drive to combine these, but they should all be separated into tests for borderRadius and borderWidth

File: framer/Animation.coffee [hunk 3]
```diff
@@ -338,7 +360,7 @@ class exports.Animation extends BaseClass
 				animatableProperties[k] = v
 			else if Color.isValidColorProperty(k, v)
 				animatableProperties[k] = new Color(v)
-			else if k is "gradient"
+			else if k in ["gradient", "borderRadius", "borderWidth"]
```
Comment (nvh) [inline framer/Animation.coffee]: Why aren't we adding these to @isAnimatable?

Comment (nvh) [inline framer/Animation.coffee]: Ah, that only gets the value. Maybe an isAnimatableValue and isAnimatableKey?

File: framer/Animation.coffee [hunk 4]
```diff
@@ -292,6 +296,24 @@ class exports.Animation extends BaseClass
...
+
+		for propKey in propKeys
+			keyValueA = if _.isNumber(valueA) then valueA else valueA[propKey]
+			keyValueB = if _.isNumber(valueB) then valueB else valueB[propKey]
+			keyValueA ?= keyValueB
+			keyValueB ?= keyValueA
+			result[propKey] = Utils.mapRange(value, 0, 1, keyValueA, keyValueB)
+
+		if _.uniq(_.values(result)).length is 1
+			result = result[propKeys[0]]
```
Comment (nvh) [inline framer/Animation.coffee]: There's a lot happening here that isn't directly clear. Maybe the naming of variables could be better, maybe it needs some comments.

File: extras/Studio.framer/framer/version (hunk 1 above)
Comment (eelco) [inline extras/Studio.framer/framer/version]: This is just the template version, so the Studio.framer project can be used with the beta. Does not really belong in this PR, sorry.

File: framer/Animation.coffee (hunk 3 above)
Comment (eelco) [inline framer/Animation.coffee]: Nice one, it’s getting unwieldy indeed.
